src/transformStateWithClones.js

```diff
@@ -7,7 +8,45 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const array = [];
+
```

Comment:
```suggestion
  const array = [];
  const stateCopy = { ...state }; 

```
Сделай сразу копию, и работй с ней